smi2ass.py

```diff
@@ -27,15 +27,14 @@
 major = sys.version_info.major
 minor = sys.version_info.minor
 
-if major == 3 and minor == 7:
+if major == 3 and minor >= 7:
     import html
 elif major == 3 and minor < 7:
     from html.parser import HTMLParser
 else:
     print ('version 3.x needed') 
 
-default_lang_code = 'kor'
```

Comment:
Could you let me know why this had to be removed?

Reply:
I thought if the subtitle file only incused single language in the subtitle it is meaningless to add kor on the file name. Since already know what language is in the subtitle file.